tests/convergence/ex1c.cpp

```diff
@@ -0,0 +1,414 @@
+//                                MFEM Example 1 - complex
+//
+// Compile with: make ex1c
+//
+//  ./ex1c -m ../../data/inline-hex.mesh -o 1 -ref 4 -sol 1 -iprob 1 -no-vis
+
+#include "mfem.hpp"
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+using namespace mfem;
+
+enum prob {comp,rl,im};
+prob prob_kind;
+int dim;
+int sol=1;
+
+double u_exact_re(const Vector &x);
+double u_exact_im(const Vector &x);
+void u_grad_exact_re(const Vector &x, Vector &du);
+void u_grad_exact_im(const Vector &x, Vector &du);
+double f_exact_re(const Vector &x);
+double f_exact_im(const Vector &x);
+void get_solution_re(const Vector &x, double & u, double du[], double & d2u);
+void get_solution_im(const Vector &x, double & u, double du[], double & d2u);
+
+int main(int argc, char *argv[])
+{
+   // 1. Parse command-line options.
+   const char *mesh_file = "../../data/star.mesh";
+   int order = 1;
+   bool static_cond = false;
+   bool visualization = true;
+   int ref_levels = 0;
```

Comment:
A default value of 1 for `ref_levels` would probably be more appropriate.

Reply:
Done